game/CharacterSheet/CharacterSheet.gd

```diff
@@ -154,22 +161,25 @@ func _on_CloseButton_button_up():
 	self.mouse_filter = Control.MOUSE_FILTER_STOP
 
 
-func _on_Ethnicity_ethnicity_chosen(trait_button, ethnicity, attribute):
-	var trait_name
+func _on_Ethnicity_ethnicity_chosen(ethnicity):
+	self.character_stats_element.ethnicity = ethnicity["ethnicity_name"]
+
+
+func _format_trait_name(trait_button):
```

Comment:
ethnicity trait

Reply:
lets leave like that - its local anyway.